.travis.yml

```diff
@@ -19,17 +19,12 @@ jobs:
         - bash -c 'shopt -s globstar; shellcheck **/*.sh'
     - &build
       stage: build
-      env: VERSION=0.17
+      env: VERSION=0.17.36 EXTRA_TAG=latest
```

Comment:
Does my autoudpater need to update these values too?
The same way the readme is updated.

Reply:
If we don't need the enviroment variable to be persistent we could replace it with https://docs.docker.com/engine/reference/commandline/build/#set-build-time-variables---build-arg .

Reply:
if you have a better idea let me know.

Normally I would spin up the container and use the returned value from the program to tag it. Something like ``VERSION=$(docker --rm dtandersen/docker_factorio_server ./factorio --version) but cause we replaced the entrypoint this is not easily possible.

Or I just use the latest tag? -_- that would probably easier.